### PR TITLE
{bio}[GCC/7.3.0-2.30] wtdbg2 v2.3

### PR DESCRIPTION
**Pull request by zao**
(created using `eb --new-pr`)

### CODE REVIEW

File: easybuild/easyconfigs/w/wtdbg2/wtdbg2-2.3-makefile-hardcoding.patch [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+ else
+-CFLAGS=-g3 -W -Wall -Wno-unused-but-set-variable -O4 -DTIMESTAMP="$(TIMESTAMP)" -D_FILE_OFFSET_BITS=64 -D_GNU_SOURCE -mpopcnt -msse4.2
++CFLAGS+=-g3 -W -Wall -Wno-unused-but-set-variable -O4 -DTIMESTAMP="$(TIMESTAMP)" -D_FILE_OFFSET_BITS=64 -D_GNU_SOURCE
+ endif
+ 
+ GLIBS=-lm -lrt -lpthread -lz
+Only in wtdbg2-2.3: pgzf
+Only in wtdbg2-2.3: wtdbg2
+Only in wtdbg2-2.3: wtdbg-cns
+Only in wtdbg2-2.3: wtpoa-cns
```
Comment (akesandgren) [inline easybuild/easyconfigs/w/wtdbg2/wtdbg2-2.3-makefile-hardcoding.patch]: Please drop these "Only in" lines from the patch

**Review by akesandgren — DISMISSED** [review]

**Comment by phineas42** [comment]
Thank you for this pr. I've successfully deployed wdtbg2 in my environment with this pr and the suggested patch modifications from @akesandgren .
(new checksum on the patch was: 'ba2d25fd3fb84b3b509c7478199ce340c70a95f0f4898c5fae46e3c74c55b793' )

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node2304.phanpy.os - Linux centos linux 7.9.2009, x86_64, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz (haswell), Python 3.6.8
See https://gist.github.com/70b1620aeff0acf6aa0edf4067e865b7 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node3125.skitty.os - Linux centos linux 7.9.2009, x86_64, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 3.6.8
See https://gist.github.com/b9e6c3cfb471d2ea20bdf1106aca9bc3 for a full test report.

**Review by boegel — APPROVED** [review]
lgtm

**Comment by boegel** [comment]
Going in, thanks @zao!

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node3507.doduo.os - Linux RHEL 8.2, x86_64, AMD EPYC 7552 48-Core Processor (zen2), Python 3.6.8
See https://gist.github.com/15770747efe51b42693749f1f03f1e4e for a full test report.

**Pull request merged**